chainservice/builder.go

```diff
@@ -344,6 +348,15 @@ func (builder *Builder) buildContractStakingIndexer(forTest bool) error {
 		return err
 	}
 	builder.cs.contractStakingIndexer = indexer
+	// indexer v2
+	dbConfig = builder.cfg.DB
+	dbConfig.DbPath = builder.cfg.Chain.ContractStakingIndexDBPathV2
```

Comment:
only do this when `len(ContractStakingIndexDBPathV2) > 0`?

Reply:
okay, moved to a separate build function